.dependency-cruiser.js

```diff
@@ -0,0 +1,397 @@
+/** @type {import('dependency-cruiser').IConfiguration} */
+module.exports = {
```

Comment:
Is this copied from somewhere - if yes, can we add a reference to that link in this file as comment?